java/client/src/test/java/glide/api/models/ClusterTransactionTests.java

```diff
@@ -57,6 +63,57 @@ public void transaction_builds_protobuf_request() {
                         Info,
                         ArgsArray.newBuilder().addArgs(InfoOptions.Section.EVERYTHING.toString()).build()));
 
+        Map<String, Double> membersScores = new HashMap<String, Double>();
+        membersScores.put("member1", 1.0d);
+        membersScores.put("member2", 2.0d);
```

Comment:
You can use the `Map.of` call to create an immutable map easily: 
```
Map<String, Double> membersScores = Map.of("member1", 1.0d, "member2", 2.0d);
```

Reply:
This map isn't immutable though. We mutate it later.

Reply:
Okay, I tried that and I remember why this doesn't work. Immutable maps don't have a defined order and this makes the test results non-deterministic and causes these tests to pass some runs and fail on other runs. I will instead add a comment about this so nobody accidentally decides to refactor this and introduce flaky tests into our codebase.

Reply:
`HashMap` also doesn't guarantee the order. `Map.of` creates a `HashMap` under the hood.
Use `LinkedHashMap` to ensure the order.